### PR TITLE
Update warpdrive tutorial

### PR DESCRIPTION
**Pull request by kaushikb11**
## Before submitting

- [ ] Was this **discussed/approved** via a Github issue? (no need for typos and docs improvements)
- [x] Did you make sure to **update the docs**?
- [x] Did you write any new **necessary tests**?

## What does this PR do?

Fixes # (issue)

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: lightning_examples/warp-drive/multi_agent_rl.py [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# %%
+import sys
+
+if "google.colab" in sys.modules:
+    ! pip uninstall --quiet "torchaudio" "torchvision" "torchtext" "fastai" "datascience" "kapre" --yes
+    ! pip install --quiet "rl-warp-drive>=1.6.5" "pytorch-lightning>=1.4"
+else:
+    ! pip install --quiet "ipython[notebook]" "rl-warp-drive>=1.6.5" "ffmpeg-python" "setuptools==59.5.0" "torch>=1.8" "torchmetrics>=0.7" "pytorch-lightning>=1.4"
```
Comment (Borda) [inline lightning_examples/warp-drive/multi_agent_rl.py:8]: this is not valid python, if you have some additional comment on how to install it on different env such as colab, lest add it as text celll

Comment (kaushikb11) [reply lightning_examples/warp-drive/multi_agent_rl.py:8]: We could use `subprocess.run(..)`

Comment (Borda) [reply lightning_examples/warp-drive/multi_agent_rl.py:8]: this is not the point, you try to band the system for this specific notebook and specific env of Colab...

**Review by Borda — CHANGES_REQUESTED** [review]

File: lightning_examples/warp-drive/multi_agent_rl.py (hunk 1 above)
Comment (Borda) [inline lightning_examples/warp-drive/multi_agent_rl.py:8]:
```suggestion
```

File: lightning_examples/warp-drive/.meta.yml [hunk 2]
```diff
@@ -18,8 +18,5 @@ description: This notebook introduces multi-agent reinforcement learning (MARL)
     documentation - http://opensource.salesforce.com/warp-drive/, and
     white paper - https://arxiv.org/abs/2108.13976.
```
Comment (Borda) [inline lightning_examples/warp-drive/.meta.yml:20]:
```suggestion

requirements:
  - rl-warp-drive>=1.6.5
  - ffmpeg-python
  # todo: after merging #155 we will relax this just to `torch<=1.10` and drop TV, TT, etc.
  - torch==1.10.*
  - torchvision==0.11.*
  - torchtext==0.11.*
```

**Review by Borda — APPROVED** [review]

**Pull request merged**